crates/sui-types/src/bridge.rs

```diff
@@ -227,7 +236,16 @@ pub struct MoveTypeBridgeTreasury {
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct MoveTypeBridgeCommittee {
     pub members: VecMap<Vec<u8>, MoveTypeCommitteeMember>,
-    pub thresholds: VecMap<u8, u64>,
+    pub stake_thresholds_percentage: VecMap<u8, u64>,
+    pub member_registrations: VecMap<SuiAddress, MoveTypeCommitteeMemberRegistration>,
+    pub last_committee_update_epoch: u64,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct MoveTypeCommitteeMemberRegistration {
+    pub sui_address: SuiAddress,
+    pub bridge_pubkey_bytes: Vec<u8>,
+    pub http_rest_url: Vec<u8>,
```

Comment:
I will add a test case for bridge_api in previous PRs so we can be more careful to not break anything here